ratis-examples/src/test/java/org/apache/ratis/examples/filestore/FileStoreStreamingBaseTest.java

```diff
@@ -72,9 +77,30 @@ public void testFileStoreStream() throws Exception {
 
     final CheckedSupplier<FileStoreClient, IOException> newClient =
         () -> new FileStoreClient(cluster.getGroup(), getProperties(), raftPeer);
-    // TODO: configurable buffer size
-    final int bufferSize = 10_000;
-    testSingleFile("foo", SizeInBytes.valueOf("2M"), bufferSize, newClient);
+
+    testSingleFile("foo", SizeInBytes.valueOf("2M"), 10_000, newClient);
+    testSingleFile("foo", SizeInBytes.valueOf("2M"), 1000, newClient);
+    testSingleFile("foo", SizeInBytes.valueOf("20M"), 100_000, newClient);
```

Comment:
write the same file three times ?

Reply:
My impression was the RandomAccessFile will overwrite existing file. 

After some research, it seems that when there is an existing file, RandomAccessFile only overwrites for the top of the file and leaves the rest of it.

So I have updated to use different file names to make the test more robust.